# INTRODUCTION  
During hominin evolution, it has been suggested that the introduction of meat into the hominin diet provided a new, calorically rich food source that could accommodate the increased energetic demands of a larger brain. Human brains have intense energetic demands (~20-25% of resting metabolic rate) compared to the average primate (~9%) and the average mammal (~3-5%) (Leonard, 2007) and the caloric surplus produced from a dietary transition towards more regular meat eating may have facilitated human brain evolution. Although humans present an extreme example of brain size, it should be noted that, generally, primates are more encephalized than the average mammal. Indeed, across primates, diet and brain size appear to be tightly linked, although researchers debate the dietary drivers of this generalized pattern of increased brain size across primates. Recent work suggested that lipid-dense foods can support encephalization in primates (Izar, 2022), while other research has shown that frugivorous primates have larger brains and that fruit intake is a dietary driver of the general encephalized pattern observed across primates (DeCasien, 2017). However, it would be a mistake to overlook primates' relationship with meat, particularly in regard to human evolution. 
In this paper, the term “meat” refers to prey such as birds, lizards, frogs, or small mammals. Insectivory, although common across primates, is beyond the scope of this research and is not included in this discussion of meat. There is a lot of variation across primates in terms of which primates will eat meat, the frequency of meat-eating behaviors and the species of prey. While some primates appear to be faithful frugivores or folivores, many primates will eat meat when presented with an opportunity. Carnivory occurs across the primate order and can be observed in all major primate taxonomic groupings (Watts, 2020). However, humans are the most frequent meat eaters and, given the hyper-derived nature of the human brain, it is likely that another dietary transition needed to occur to facilitate the caloric demands of such a brain expansion in humans beyond the dietary drivers which underpin the general trend of primate encephalization. 
While humans do eat meat with more regularity than many other primates, I argue that human meat eating is not only a difference in degree, but most importantly a difference in kind. Humans can more completely exploit the animal remains through tool use, such as the extraction of lipid-rich bone marrow. Indeed, early humans participated in a unique dietary niche of meat exploitation and it appears this dietary incorporation of meat preceded human brain expansion (Ferraro, 2013). The degree of meat exploitation and the additional processing of bone marrow appear to be unique features of the human dietary niche and, I hypothesize, that this enhanced dietary inclusion of meat proteins/lipids was a primary diver in supporting human brain expansion. To evaluate the importance of meat relative to other food sources in hominin evolution, this project compares genes related to the taste perception and digestion of protein across primate lineages in order to evaluate selection on these genes within the hominin lineage. Specifically, this class project explores two genes, TAS1R1 and TAS1R3, which form the umami taste receptor heterodimer complex. This taste receptor has been connected to the perception of “meaty” or “umami” flavor in humans and, if meat was a critical component in hominin evolutionary history, the genes which encode these receptors may have experienced positive selection during hominin evolution. 

## Umami Receptor Genes
While there have been several other genes implicated in the perception of umami flavor, much research has focused on the TAS1R1 and TAS1R3 genes, which form a T1R1/T1R3 heterodimer responsible for the transduction of umami flavor. Umami flavor is described as the "savory" component to foods and is associated with the flavors of cooked meats, specific cheeses (Diepeveen, 2022) and fermented foods (Diepeveen, 2022; Zhang, 2019). This receptor is a G protein-coupled receptor (GPCR) which, in humans, is activated by binding of monosodium glutamate (MSG), free L-amino acids and peptides (Zhang, 2019). However, research indicates the substances to which these receptors respond vary across primates. For example, Todo et al. cloned TAS1R1 receptors of numerous primates across the primate phylogeny and exposed these receptors to L-glutamate (L-glu) and 5’ ribonucleotides. They found folivorous primate receptors were responsive to L-glu, which is a prominent component of leaves, and insectivorous primate receptors were responsive to 5' ribonucleotides, which is a prominent component of insects. The receptors of primates with more diverse diets, termed dietary generalists, were responsive to both L-glu and 5' ribonucleotides (Toda, 2021). This research is important as it illustrates critical limitations within this project: TAS1R1 serves purposes beyond meat perception in other primates and, thus, this complicates the examination of these genes in the hominin lineage compared to other primate lineages. The function of TAS1R3,  however, is less clear across primates. While this complicated history must be considered, exploration of the TAS1R1 and TAS1R3 is worthwhile as research links these genes to meat flavor perception in humans and, while their history may be complicated across primates, these genes may still be critical within the hominin lineage specifically.  

## Hypothesis Testing in PAML 
Phylogenetic Analysis by Maximum Likelihood (PAML) is a package for phylogenetic analyses and its program CODEML can be used to evaluate selection. CODEML is based on a codon model which considers the transition/transversion rate ratio (k) and nonsynonymous/synonymous rate ratio (ω = dN/dS) (Yang, 1998; Alvarez-Carretero, 2023). The key to successful PAML use lies in defining a clear biological question and selecting the appropriate model(s) for this hypothesis. Models fall into three broad categories: branch models, site models and branch-site models. Branch models allow ω to vary across branches. The free ratio model (CODEML settings: model=1, NSsites=0) is a type of branch model that allows each branch to have its own ω (Yang 1997). The free ratio model is very parameter rich but can be useful in evaluating the heterogeneity of ω across branches. To test for selection in specific branches of the phylogeny, one can use the branch model with specified foreground branches (CODEML settings: model=2, NSsites=0) where only the branches denoted as foreground branches have variable ω ratios and background branches are held constant. To test for selection in specific sites, the site model (CODEML settings: model=0, NSsites = 1 2 7 8) may be used. The branch-site model A (CODEML settings: model=2, NSsites=2) allows ω to vary across sites and branches. The branch-site model can be prone to false positives. The one rate ratio model (also known as M0 and homogenous model; CODEML settings: model=0, NSsites=0) assumes one ω ratio for all sites and branches. This model is not realistic as many sites are under purifying selection, but this model can 1) indicate the level of selective constraint and 2) serve as a null model when comparing across other models for best fit (Alvarez-Carretero, 2023). 
This project is specifically concerned with selection occurring in branches, therefore, the branch model was most appropriate. The one-rate ratio model was used for the null model. The hypotheses are as follows: 

If hominins underwent a dietary transition towards meat eating which was critical to supporting an energetically expensive brain, it is possible that the taste receptor genes critical for the taste perception of meat may have experienced selection. 

**H0a: The TAS1R1 and/or TAS1R3 genes do not show signatures of selection in the hominin lineage.
H1: TAS1R1 and/or TAS1R3 show positive selection in the hominin lineage as human meat-eating behavior is different in kind from other primates.**

While humans are the most prolific meat eaters, their close relatives chimpanzees (Pan troglodytes) and bonobos (Pan paniscus) are known to participate in carnivorous behaviors at greater frequencies than many other primates. As such, it is possible that there may be selection on genes related to meat consumption in the shared branch which leads to Homo and Pan.

**H0b: The TAS1R1 and/or TAS1R3 genes do not show signatures of selection in both Homo and Pan species. 
H2: TAS1R1 and/or TAS1R3 show positive selection in H. sapiens, P. troglodytes and P. paniscus as meat consumption is important in this clade relative to other primates.**   

# METHODS
## Evaluation of Gene History
Both TAS1R1 and TAS1R3 were examined on ensembl.org. For both genes, all primate species had 1:1 orthologue status, meaning only one copy was found in each of the species. To further examine the possibility of gene duplication events or any other complexity of gene history confounding analyses of these genes, I separately ran both the TAS1R1 and TAS1R3 human amino acid transcripts through BlastP to check the sequences that returned with high similarity. For both genes, all returning sequences were labeled as the gene of interest. There did not appear to be any other sequences returning. Notably, some species did have multiple isoforms of the gene. I individually checked each primate from Ensembl.org and some species did have 2-4 known splice variants for either gene, so these results were not surprising.

## Sequence Acquisition & Alignment 
DNA and amino acid sequence data for the TAS1R1 gene (ENSG00000173662) and the TAS1R3 gene (ENSG00000169962) were gathered from ensembl.org on February 8th, 2023. Ensembl identified 22 species of primates with orthologous sequences for TAS1R1 and 19 species of primates with orthologous sequences for TAS1R3. Both the amino acid sequence and the DNA sequence were downloaded. First, they were aligned using muscle, then, all gaps in the DNA were checked and manually adjusted based on the amino acid alignment to ensure a proper reading frame of codons. The TAS1R1 alignment was noted to be of high quality with only Neumascus leugenys requiring adjustment. The TAS1R3 alignment showed more mutations between species. Particularly, Roxellana bieti showed numerous insertions and deletions throughout the alignment. Although the subsequent steps were performed with Roxellana in the TAS1R3 alignment, future re-analyses may remove this species to see if it impacts the results, as this primate appears to be an extreme outlier. Aside from Roxellana, the TAS1R3 alignment was of good quality. 

## Phylogenetic Tree Generation
Following DNA alignment, the alignments for TAS1R1 and TAS1R3 were input in PAUP and both underwent a Maximum Likelihood bootstrap analysis (500 replicates). The resultant trees showed that all primates grouped together based on relatedness for both genes (i.e. New World Monkeys formed a clade, Old World Monkeys formed a clade, etc). The majority of bootstrap values were greater than 95 with the exception of four values for TAS1R1 (52.2, 54, 76.8, and 86.6) and four values for TAS1R3 (58.6, 60.4, 63.8 and 81.6). Additionally, the TAS1R1 and TAS1R3 alignment files were combined and then input together into PAUP, where they underwent a Maximum Likelihood bootstrap (500 replicates) as described above. The prosimian TAS1R1 gene was the outgroup. Following this, the TAS1R1 and TAS1R3 genes split and grouped via relatedness.
During the first run, only consensus sequence trees were obtained. This is not sufficient for PAML input. I reran the alignments for both TAS1R1 and TAS1R3 in order to generate phylogenies. During this second attempt, PAUP’s command line was used as opposed to its GUI in order to aid in reproducibility measures. The phylogenies generated from this second run of TAS1R1 and TAS1R3 were used in the subsequent analyses. 

## PAML Analysis 
The TAS1R1 and TAS1R3 genes underwent identical protocols (general protocols evaluated from Álvarez-Carretero 2023, Bielawski 2013, Bielawski 2016, Yang 1997, and Yang 1998). A one-rate ratio model was ran to evaluate levels of constraint and to establish a null model. This model was set with the following settings: model=0, NSsites=0. A branch model was used for both of the alternative hypotheses with the following settings: model=2, NSsites=0. Under the alternative hypothesis that selection occurred on the human branch in contrast to others, the human branch was selected as the foreground. Under the second alternative hypothesis that selection occurred on the branch leading to both Homo and Pan, there was a difference between the two genes in how the foreground branch was set (see Image2 and Image3 in the appendix to see the phylogenetic trees of TAS1R1 and TAS1R3 respectively). In TAS1R1, the Gorilla gorilla TAS1R1 gene was more closely related to the Pan TAS1R1 gene than the Homo TAS1R1 gene, likely due to incomplete lineage sorting. As a result, the Pan branch was selected as the foreground and the hominin branch was also selected as the foreground to evaluate the unlikely possibility that TAS1R1 may have experienced convergent selection in Pan and Homo to the exclusion of Gorilla. However, in TAS1R3, both Homo and Pan formed a clade and the branch for both Homo and Pan was marked as the foreground during the TAS1R3 alternative hypothesis 2 run. Additionally, all hypotheses were run under two different codon frequency models. They were first run under F1X4 (CodonFreq=2) and then rerun under FMutSel (CodonFreq=7), which is thought to be more sensitive to selective processes (Álvarez-Carretero, 2023). Upon completion of PAML runs, the number of parameters and the log-likelihood scores were extracted from the results and input into R where the alternative hypotheses underwent a likelihood ratio test against the null model. 

# RESULTS AND REFLECTION
## Results 
TAS1R1 and TAS1R3 were evaluated for selection in the human branch as well as in the branch for both human and Pan species. In TAS1R1, there was no support for either of the alternative models and the null was not rejected (results in Table 1). When contextualized into the broader framework of TAS1R1 research, this may not be surprising. Research has shown that TAS1R1 acts functionally different in other primates in accordance with their dietary specialization and is important in diets ranging from folivory to insectivory. As a result of this complicated evolutionary history, it was unlikely that TAS1R1 would experience greater selection in either the human branch or the Homo/Pan clade relative to other primates. 

For TAS1R3, while the null was not rejected for selection in the hominin lineage only, there was support for the selection occurring in the Homo/Pan clade (p=.0054 under F1X4; p=.0057 under FMUTSEL; results in Table 1). This result is interesting as it contradicts ideas that meat serves a uniquely important role in the Homo genus. Instead, meat consumption may be a critical dietary component for both Homo and Pan. The meat-eating behaviors of Pan have typically been described as relatively infrequent and opportunistic, yet this type of framing could obscure the critical importance of this food source in our shared evolutionary history. Indeed, although further research on dietary adaptations to meat needs to be conducted, if there is a continued pattern of selection in both Homo and Pan as opposed to exclusively Homo, then discussions of the critical importance of meat consumption in the hominin lineage specifically may be too general and over-simplistic and, perhaps, there are specific nutrient profiles which carry more explanatory power within the context of hominin brain expansion. 

## Produced Image 
I had initially anticipated creating a phylogenetic tree for my results as shown as Image 1 in the appendix. This tree was created based on results from a branch-site model and contained information specific to that project, such as discussion of sleep patterns. Given that the design of my project was very different, I opted to create a phylogenetic tree specific to my research and aimed to best showcase my results while employing some features from the inspiration image, such as the clade labeling. However, as my analysis was specifically interested in only Homo and Pan, I decided to keep the visual clutter minimal and only label the branches with species names for the foreground branches and the species which were closely related. The phylogenetic trees for both TAS1R1 and TAS1R3 can be viewed as Image 2 and Image 3 respectively in the appendix. They are captioned with a description of the results and the motives behind aesthetic choices. 

## Reproducibility 
For this project, I have tried to enhance reproducibility wherever possible, such as switching from using PAUP’s GUI to using it through the command line on the second run. By making this switch, there is a file that contains the exact parameters used as opposed to the GUI where I would have had to self-report. Additionally, I worked to consistently update README.md files and make sure they contained clear information about versions, dates, and directory contents. I intend to continue to improve my reproducibility and plan to integrate checksums regularly into my README.md files, stay more diligent about annotating R scripts and create a research project “protocol” that lays out a very regimented style of file labeling and documentation to improve research flow over the summer. 

## Reflection 
The greatest challenge associated with this project was properly using the PAML package, which is quite complicated and requires a good foundational knowledge in order to properly use it and interpret its output. When I proposed this project at the beginning of the semester, I had an incomplete understanding of the PAML package. I had only used it a handful of times in test runs to see how to set foreground branches and try to understand its input parameters. Additionally, while I had read many papers which used PAML, I did not entirely understand the appropriate way to test hypotheses with this package. At the time of the proposal, I knew this would be a challenging project for me but I am happy with the progress I made this semester in conceptually understanding CODEML and that I was able to design a project using the software and complete an analysis. With that said, I still see limitations in my understanding of the package. Moving forward this summer, I intend to finish the book Computational Molecular Evolution by Ziheng Yang, who is the creator of PAML, as well as read more of the papers by Yang to continue to fill out my conceptual understanding of PAML’s input parameters, proper hypothesis testing in PAML, etc. 

Additionally, I intend to expand the scope of my analysis to include more genes involved in meat perception and digestion beyond the  TAS1R1 and TAS1R3 genes. This is work that I aim to continue this summer. Given existing research linking fruit and brain expansion across primates, it may be worthwhile to explore genes related to both protein and fruit digestion. It could be interesting to see if more meat-related genes experience selection in the hominin lineage as opposed to fruit-related genes, and also evaluate if the inverse trend appears in other primates. 


APPENDIX
![Image 1. Inspiration image from Wu et al. As my research project progressed, its research design deviated from the design employed in this research project. As a result, I chose what I thought were the most effective design elements of this image and used them in my images.](Homework0_PrimatePhylogenyImage.jpg)

![Table 1. The results generated from the Likelihood Ratio Test in R. The naming convention is as follows: 1) f1x4 vs fmutself indicates which codon frequency model the alternative hypothesis used, 2) hum vs pan indicates if the foreground branch was human only (hum) or homo/pan clade (pan), 3) all are labeled alt as they were the alternative hypothesis, 4) the last label indicates which gene was tested.](results_image.jpg)

![Image 2. The below tree was produced in R using ggtree. For this gene, the null hypothesis was not rejected. As a result, this image showcases the branches tested and is subtitled to clearly state that selection was not detected in this analysis. Aesthetics: The color red was chosen to further represent the lack of null rejection. As stated above, the clade labels were used to show general groups but I chose not to provide the label names of all species in order to avoid visual clutter. While the example image used nodes to show selection, I opted to shade as only a couple nodes would look odd.](tas1R1.jpg)

![Image 3. The below tree was produced in R using ggtree. The null hypothesis was rejected and there is support for selection in the Homo-Pan clade, but not in only Homo. Aesthetics: I chose to highlight this clade and chose the color green to represent these results. As stated above, the clade labels were used to show general groups but I chose not to provide the label names of all species in order to avoid visual clutter. While the example image used nodes to show selection, I opted to shade as only a couple nodes would look odd. ](tas1r3.jpg)


REFERENCES
Álvarez-Carretero, S., Kapli, P., & Yang, Z. (2023). Beginner's guide on the use of PAML to detect positive selection. Molecular Biology and Evolution, 40(4), msad041.

Bielawski, J. P. (2013). Detecting the signatures of adaptive evolution in protein‐coding genes. Current Protocols in Molecular Biology, 101(1), 19-1. DOI: https://doi.org/10.1002/0471142727.mb1901s101.

Bielawski, J. P., Baker, J. L., & Mingrone, J. (2016). Inference of episodic changes in natural selection acting on protein coding sequences via CODEML. Current protocols in bioinformatics, 54(1), 6-15. DOI: https://doi.org/10.1002/cpbi.2.

DeCasien, A. R., Williams, S. A., & Higham, J. P. (2017). Primate brain size is predicted by diet but not sociality. Nature ecology & evolution, 1(5), 0112.DOI: https://doi.org/10.1038/s41559-017-0112. 

Diepeveen, J., Moerdijk‐Poortvliet, T. C., & van der Leij, F. R. (2022). Molecular insights into human taste perception and umami tastants: A review. Journal of Food Science, 87(4), 1449-1465. DOI: https://doi.org/10.1111/1750-3841.16101.

Ferraro, J. V., Plummer, T. W., Pobiner, B. L., Oliver, J. S., Bishop, L. C., Braun, D. R., ... & Potts, R. (2013). Earliest archaeological evidence of persistent hominin carnivory. PloS one, 8(4), e62174. DOI:10.1371/journal.pone.0062174.

Izar, P., Peternelli-dos-Santos, L., Rothman, J. M., Raubenheimer, D., Presotto, A., Gort, G., ... & Fragaszy, D. M. (2022). Stone tools improve diet quality in wild monkeys. Current Biology, 32(18), 4088-4092. DOI: https://doi.org/10.1016/j.cub.2022.07.056.

Leonard, W. R., Snodgrass, J. J., & Robertson, M. L. (2007). Effects of brain evolution on human nutrition and metabolism. Annu. Rev. Nutr., 27, 311-327. DOI: 10.1146/an- nurev.nutr.27.061406.093659.

Toda, Y., Hayakawa, T., Itoigawa, A., Kurihara, Y., Nakagita, T., Hayashi, M., ... & Misaka, T. (2021). Evolution of the primate glutamate taste sensor from a nucleotide sensor. Current Biology, 31(20), 4641-4649. DOI:https://doi.org/10.1016/j.cub.2021.08.002. 

Watts, D. P. (2020). Meat eating by nonhuman primates: a review and synthesis. Journal of human evolution, 149, 102882.DOI:Https://doi.org/10.1016/j.jhevol.2020.102882. 

Wu, Y., Wang, H., Wang, H., & Hadly, E. A. (2017). Rethinking the origin of primates by reconstructing their diel activity patterns using genetics and morphology. Scientific Reports, 7(1), 11837. DOI: https://doi.org/10.1038/s41598-017-12090-3. 

Zhang, J., Sun-Waterhouse, D., Su, G., & Zhao, M. (2019). New insight into umami receptor, umami/umami-enhancing peptides and their derivatives: A review. Trends in Food Science & Technology, 88, 429-438. DOI: https://doi.org/10.1016/j.tifs.2019.04.008.

Zhang, J. (2006). Parallel adaptive origins of digestive RNases in Asian and African leaf monkeys. Nature genetics, 38(7), 819-823.DOI: https://doi.org/10.1038/ng1812.

Yang, Z. (1997). PAML: a program package for phylogenetic analysis by maximum likelihood. Computer applications in the biosciences, 13(5), 555-556. DOI: 10.1093/bioinformatics/13.5.555. 

Yang, Z. (1998). Likelihood ratio tests for detecting positive selection and application to primate lysozyme evolution. Molecular biology and evolution, 15(5), 568-573. https://doi.org/10.1093/oxfordjournals.molbev.a025957.
